.github/workflows/test.yml

```diff
@@ -101,7 +101,29 @@ jobs:
       - run: bash scripts/environment/prepare.sh
       - run: echo "::add-matcher::.github/matchers/rust.json"
       - run: make slim-builds
-      - run: make test-components
+      - run: make test
+
+  # Remove this once https://github.com/timberio/vector/issues/3771 is closed.
+  # Then, modify the `cross-linux` job to run `test` instead of `build`.
+  test-benches-linux:
```

Comment:
I think we need to add these new jobs to the notify job at the bottom.

Reply:
Quite right. I did miss that.